rhel7/profiles/stig.profile

```diff
@@ -252,7 +252,7 @@ selections:
     - sshd_use_priv_separation
     - sshd_disable_compression
     - chronyd_or_ntpd_set_maxpoll
-    - configure_firewalld_rate_limiting
```

Comment:
The ``configure_firewalld_rate_limiting`` rule should not be removed. This differs by performing a rate limit of connections per second.

Reply:
(note: it is possible that ``configure_firewalld_rate_limiting`` is no longer required by DISA, in which case yes, this can/should be removed from the stig profile)

Reply:
If this is the latest STIG for Rhel7, then I do not see any finding represented by configure_firewalld_rate_limiting there.
https://stigviewer.com/stig/red_hat_enterprise_linux_7/